config/locales/en.yml

```diff
@@ -1055,6 +1053,13 @@ en:
         max_res: "Maximum Reservation Minutes"
         cancel_hours: "Minimum Cancelation Hours"
         cancel_mins: "Automatic Cancelation Minutes"
+    offline:
+      note: Instrument is down
+      notice: |
+        This instrument is currently down.
+        Contact facility for more information.
+        It will be canceled for $0 if the instrument is still down at the time
```

Comment:
I initially had it as "This instrument is currently offline. If it is not back online before your scheduled time, you will receive a notice and your reservation will be canceled" but the wording in 2a07e1ce295cd8c67c65cda39b4f6e029a7130fd came from the ticket. I'll rewrite.